modules/openy_gc_auth/modules/openy_gc_auth_yusa/src/YUSAClientService.php

```diff
@@ -18,7 +18,7 @@ class YUSAClientService {
    *
    * @var \Drupal\Core\Logger\LoggerChannelFactory
```

Comment:
It will be great to correct the class or interface here

Reply:
It will be great, but not today! :stuck_out_tongue: